utilities/copyoffload_migration.py

```
@@ -205,11 +205,7 @@ def _check_file() -> bool:
 
     finally:
         if target_power_state == "off":
-            LOGGER.info(f"Powering off VM - {vm_name}")
-            try:
-                source_provider.stop_vm(provider_vm_api)
-            except Exception as e:
-                LOGGER.warning(f"Failed to power off VM '{vm_name}': {type(e).__name__}: {e}")
+            LOGGER.info(f"VM {vm_name} left powered on — MTV will handle shutdown for cold migration")
```

Ruff (0.15.9)

[warning] Logging statement uses f-string

(G004)